subprojects/hse-python.wrap

```diff
@@ -1,3 +1,3 @@
 [wrap-git]
 url = https://github.com/hse-project/hse-python.git
-revision = v2.1
+revision = r2.1.0
```

Comment:
I feel like this should be r2.1

Reply:
Just following pattern from v2.0, will let @alexttx make the call here.

Reply:
Sounds good.

Reply:
Tagged hse releases use tagged hse-python releases.